packages/synchro-charts/src/testing/app/routes.ts

```diff
@@ -345,4 +345,8 @@ export const routes = [
     url: '/tests/widget-test-route',
     component: 'widget-test-route',
   },
+  {
+    url: '/tests/sc-dial',
+    component: 'sc-dial-standard',
+  },
```

Comment:
remove this and utilize 

```
  visitDynamicWidget(cy, {
    componentTag,
    dataStreams: [],
    messageOverrides,
  });
```
style instead, seen in the more recently written integration tests to avoid having to support and maintain hundreds of webcomponents

Reply:
Modify visitDynamicWidget in integration tests.